Extensions/Xtensive.Orm.Localization.Tests/Model/DictionaryModel.cs

```diff
@@ -0,0 +1,55 @@
+using System;
```

Comment:
Please add following copyright at the beginning of the file:
```
// Copyright (C) 2020 Xtensive LLC.
// This code is distributed under MIT license terms.
// See the License.txt file in the project root for more information."
```

Reply:
done